orchestra/communication/utils.py

```diff
@@ -73,3 +73,23 @@ def mark_worker_as_winner(worker, task, required_role_counter,
             request_inquiry=staffing_request_inquiry,
             is_available=True,
             is_winner=True)
+
+
+def close_open_staffbot_requests(task):
+    requests = task.staffing_requests.all()
+    for request in requests:
+        close_staffbot_request(request)
+
+
+@transaction.atomic
+def close_staffbot_request(request):
+    if request.is_open():
+        inquiries = request.inquiries.all()
```

Comment:
similar to the comment below, can we query for all inquiries that don't have responses to reduce round trips?